### PR TITLE
L10N-315 2020-11-27-1215 - translations batch

### PR DESCRIPTION
**Pull request by alexandermccate**

### CODE REVIEW

File: packages/L10n/src/locales/fr.yml [hunk 1]
```diff
@@ -102,21 +102,24 @@ fr:
...
+        a11y_text: Rechercher
         no_results_message: Vos résultats de recherche ne correspondaient à aucune option.
-        placeholder: Filtrer...
+        placeholder: Rechercher...
     listBoxBrowser:
       explore: Découvrir d'autres options
       selected: options sélectionnées
       selectOne: Sélectionner un ou plusieurs éléments...
+    listBoxWithTags:
+      placeholder: Sélectionner…
```
Comment (cristinajimenez) [inline packages/L10n/src/locales/fr.yml:113]: Regular ellipsis can be used here: `...`

File: packages/L10n/src/locales/pt.yml [hunk 2]
```diff
@@ -102,21 +102,24 @@ pt:
...
+        a11y_text: Pesquisar
         no_results_message: O resultado da sua pesquisa não correspondeu a nenhuma opção.
-        placeholder: Filtrar...
+        placeholder: Pesquisar...
     listBoxBrowser:
       explore: Explorar mais opções
       selected: opções selecionadas
       selectOne: Selecione um ou mais itens...
+    listBoxWithTags:
+      placeholder: Selecione…
```
Comment (cristinajimenez) [inline packages/L10n/src/locales/pt.yml:113]: Regular ellipsis can be used here: `...`

**Review by jamiek-acl — DISMISSED** [review]

File: packages/L10n/src/locales/pt.yml [hunk 3]
```diff
@@ -176,4 +179,6 @@ pt:
     sideNavigation:
       aria_label: Navegação lateral
       trigger_aria_label: Abrir navegação
-          
\ No newline at end of file
+    copyInput:
+      clicked_tooltip: Copiado
+      hover_tooltip: Copiara para área de transferência
```
Comment (cristinajimenez) [inline packages/L10n/src/locales/pt.yml:184]: I think this should be `Copiar`

**Review by cristinajimenez — CHANGES_REQUESTED** [review]

**Comment by alexandermccate** [comment]
Fixes applied

**Review by cristinajimenez — APPROVED** [review]

**Pull request merged**